sdk/servicebus/Azure.Messaging.ServiceBus/src/Core/ServiceBusRetryOptionsExtensions.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Messaging.ServiceBus.Core
+{
+    /// <summary>
+    ///   The set of extension methods for the <see cref="ServiceBusRetryOptions" />
+    ///   class.
+    /// </summary>
+    ///
+    internal static class ServiceBusRetryOptionsExtensions
```

Comment:
What's the benefit of making these extensions over just including them with the type?   If you feel strongly that they need to be segmented, maybe consider a partial type instead so that they're still compiled as a single class?

Reply:
This was copied over from EH Track 2.